yarn.lock

```diff
@@ -10543,10 +10538,10 @@ sugarss@^2.0.0:
   dependencies:
     postcss "^7.0.2"
 
-suomifi-design-tokens@0.3.0:
-  version "0.3.0"
-  resolved "https://registry.yarnpkg.com/suomifi-design-tokens/-/suomifi-design-tokens-0.3.0.tgz#694660b5038e1a652ad64d2a8e727f0561392588"
-  integrity sha512-brxPbFjpJUNm17SF/d174UhZtZGZJPL8OhRApBVi30eCm9BFptiqD18SLy1k58dlloVfqGjhALNnHiwhEcFftA==
+suomifi-design-tokens@0.4.0:
+  version "0.4.0"
+  resolved "https://registry.npmjs.org/suomifi-design-tokens/-/suomifi-design-tokens-0.4.0.tgz#5b8e4d0bc60b2274287c8d1bbb6990bb74d0e52f"
```

Comment:
is the registry change something that should be addressed?

Reply:
Does not currently have an effect to anything, but to keep things future proof, I'd change it back to yarnpkg. Especially in this case, because this repo has no references to npmjs currently.